src/main/java/com/example/hibernate/model/UpdateCar.java

```diff
@@ -0,0 +1,23 @@
+package com.example.hibernate.model;
+
+public record UpdateCar (int yearOfManufacture, String registerNumber, User user, Manufacturer manufacturer) {
+    @Override
```

Comment:
I think you don't need `@Override` here. I'm not even sure why this works (if it works)